Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift

```diff
@@ -23,61 +23,71 @@ final public class CharacteristicCyclingSpeedCadence: Characteristic {
     public static var uuidString: String { "2A5B" }
     
     
+    public struct Flags: OptionSet, Hashable {
+        public let rawValue: UInt8
+        public init(rawValue: UInt8) { self.rawValue = rawValue }
+        
+        /// Ringer State active
+        public static let wheelRevolutionDataPresent = Flags(rawValue: 1 << 0)
+        /// Vibrate State active
+        public static let crankRevolutionDataPresent = Flags(rawValue: 1 << 1)
+
+    }
+    
     /// Name of the Characteristic
     public var name: String { Self.name }
     
     /// Characteristic UUID String
     public var uuidString: String { Self.uuidString }
     
-    
-    public enum Flags: UInt8 {
-        case wheelPresent = 0
-        case crankPresent = 1
-    }
-    
     private(set) public var flags: Flags
```

Comment:
All the Vars need Comments to explain what they are..
This is a Framework for others so be kind and provide information